Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -116,6 +146,36 @@ YUI.add('ez-actionbarview-tests', function (Y) {
         "Should not instantiate deleteActionView": function () {
             this._isNotActionCreated('deleteContent', {});
         },
+
+        "Should disable sendToTrashActionView if root location": function() {
```

Comment:
Found it, so it's more about reusing: https://github.com/ezsystems/PlatformUIBundle/blob/306ff4b8c42ab0126bf6111fbb14f71ac393d83e/Tests/js/views/fields/assets/ez-richtext-editview-tests.js#L602